definitions/divorce/json/AuthorisationCaseField/AuthorisationCaseField.json

```diff
@@ -7102,7 +7102,7 @@
     "CaseTypeID": "DIVORCE",
     "CaseFieldID": "PreviousIssueDate",
     "UserRole": "caseworker-divorce-solicitor",
-    "CRUD": "R"
+    "CRUD": "CRU"
   },
```

Comment:
Hey @Dizrupto, Just a suggestion: Because of the current hoops around merging to master on AAT might be better to add this update using the prod/nonprod system. Theres alot of process to merge this in if when generating the config it in anyway changes add to the prod file definition.

Reply:
Yep that is a good point.
Did that for the other one. This one I changed directly because I saw that it's actually not used anywhere on CCD.
Will try to do the same here.